### PR TITLE
Fix #21974 Add Yasson basic test

### LINKED ISSUE #21974: Add Yasson basic test
Add Yasson test to determine if Yasson has been loaded correctly to Glassfish.

### PR DESCRIPTION
**Pull request by Verdent**
Fix for #21974

### CODE REVIEW

**Comment by Verdent** [comment]
@glassfishrobot Run CI tests please

File: appserver/tests/appserv-tests/devtests/web/yasson/YassonTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 2011-2017 Oracle and/or its affiliates. All rights reserved.
```
Comment (yaminikb) [inline appserver/tests/appserv-tests/devtests/web/yasson/YassonTest.java]: Copyright year needs to be fixed in all the new files.

File: appserver/tests/appserv-tests/devtests/web/yasson/YassonTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+ * Unit test for:
+ *      Verification that Yasson has been loaded without errors
+ */
+
+public class YassonTest {
+
+    private static SimpleReporterAdapter stat
+        = new SimpleReporterAdapter("appserv-tests");
+
+    private static final String TEST_NAME = "trasnform-to-json-by-yasson";
```
Comment (yaminikb) [inline appserver/tests/appserv-tests/devtests/web/yasson/YassonTest.java]: Typo

File: appserver/tests/appserv-tests/devtests/web/yasson/build.xml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+        <antcall target="webclient-war-common">
+            <param name="hasWebclient" value="yes"/>
+            <param name="webclient.war.classes" value="**/*.class"/> 
+        </antcall>
+
+        <javac srcdir="." classpath="${env.APS_HOME}/lib/reportbuilder.jar"
+               includes="YassonTest*"/>
+    </target> 
+    
+    <target name="enable-access-logging" depends="init-common">
```
Comment (yaminikb) [inline appserver/tests/appserv-tests/devtests/web/yasson/build.xml]: Just curious, is access logging for debugging purpose?

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Comment by Verdent** [comment]
Thank you @yaminikb for suggestions. I have implemented them in new commit.

**Comment by Verdent** [comment]
@glassfishrobot Run CI tests please

**Review by yaminikb — APPROVED** [review]

**Comment by yaminikb** [comment]
@Verdent Can you please sync your branch and re-run the tests?

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests please

**Pull request merged**